packages/block-library/src/gallery/edit.js

```diff
@@ -391,12 +391,14 @@ class GalleryEdit extends Component {
 				<InspectorControls>
 					<PanelBody title={ __( 'Gallery settings' ) }>
 						{ images.length > 1 && (
-							<ColumnsControl
+							<RangeControl
 								label={ __( 'Columns' ) }
 								value={ columns }
 								onChange={ this.setColumnsNumber }
 								min={ 1 }
 								max={ Math.min( MAX_COLUMNS, images.length ) }
+								separatorType="fullWidth"
```

Comment:
Fyi: until this lands: https://github.com/WordPress/gutenberg/pull/21365 we'll need to use the somewhat ugly `{ ...MOBILE_CONTROL_PROPS }` hack here, otherwise the `separatorType` will be rendered as an html attribute on web.

Reply:
Oh yeah thanks @mkevins I forgot about that! I’ll take care of that now

Reply:
That's adjusted now!

Reply:
Nice :+1: